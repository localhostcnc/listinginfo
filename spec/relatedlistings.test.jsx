/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';

import RelatedListings from '../dist/Components/relatedlistings';

describe('Components', () => {
  test('Rendering the Related Listings component', () => {
    const wrapper = shallow(
      <RelatedListings />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
