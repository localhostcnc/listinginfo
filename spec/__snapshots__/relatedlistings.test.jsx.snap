// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Rendering the Related Listings component 1`] = `ShallowWrapper {}`;
